backend/services/deployments/tests/tests/test_release_v2.py

```diff
@@ -119,39 +124,13 @@ def test_get_all_releases_types(self, mongo, cli):
                     ("bar", "device-type-2", "directory"),
                 ]
             ):
-                # this is a hack, since the swagger client is not prepared for the
-                # specifications of API v2 in a separate file, and we are supposed
-                # to move to openapi -- hence the fallback to requests.
-                get_types_url = (
-                    "http://"
-                    + pytest_config.getoption("host")
-                    + f"/api/management/v2/deployments/releases/all/types"
-                )
-
-                r = requests.get(
-                    get_types_url,
-                    verify=False,
-                    headers={"Authorization": "Bearer foo"},
-                )
-                types = json.loads(r.text)
+                types = management_v2_client(jwt="foo").list_release_types()
                 assert len(types) > 0
                 assert types == ["rootfs-image", "app", "single-file", "directory"]
 
     @pytest.mark.usefixtures("clean_db", "clean_minio")
-    def test_get_all_releases_types(self, mongo, cli):
+    def test_get_all_releases_types_empty(self, mongo, cli):
         with Lock(MONGO_LOCK_FILE) as l:
             cli.migrate()
-            # this is a hack, since the swagger client is not prepared for the
-            # specifications of API v2 in a separate file, and we are supposed
-            # to move to openapi -- hence the fallback to requests.
-            get_types_url = (
-                "http://"
-                + pytest_config.getoption("host")
-                + f"/api/management/v2/deployments/releases/all/types"
-            )
-
-            r = requests.get(
-                get_types_url, verify=False, headers={"Authorization": "Bearer foo"},
-            )
-            types = json.loads(r.text)
-            assert types == []
+            types = management_v2_client(jwt="foo").list_release_types()
+            assert len(types) == 0
```

Comment:
This test seems to be exclusively in open source only :monocle_face:

Reply:
it was some kind of messup. I will bring it to ent as well. but in a followup PR.